assets/js/theme/checkout/customer.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { SnackbarContent } from 'material-ui/Snackbar';
+import Button from 'material-ui/Button';
+import TextField from 'material-ui/TextField';
+import Typography from 'material-ui/Typography';
+
+export default class CustomerComponent extends React.PureComponent {
```

Comment:
I believe in most cases we can use `PureComponent` unless you have your own `shouldComponentUpdate` implementation or your states and props are not immutable. In both cases, you're not so I _think_ it's safe to use `PureComponent`.